app/models/form/sales/questions/shared_ownership_type.rb

```diff
@@ -6,7 +6,7 @@ def initialize(id, hsh, page)
     @header = "What is the type of shared ownership sale?"
     @hint_text = "A shared ownership sale is when the purchaser buys up to 75% of the property value and pays rent to the Private Registered Provider (PRP) on the remaining portion"
     @type = "radio"
-    @answer_options = ANSWER_OPTIONS
+    @answer_options = form.start_date.year >= 2023 ? ANSWER_OPTIONS_23_24 : ANSWER_OPTIONS
```

Comment:
You should make `answer_options` a method — see `Form::Common::Questions::CreatedById` for an example.